ciao-launcher/vmconfig.go

```diff
@@ -96,6 +96,14 @@ func (cfg *vmConfig) findVolume(UUID string) *volumeConfig {
 	}
 	return nil
 }
+func (cfg *vmConfig) haveBootableVolume() bool {
```

Comment:
As things stand right now, if you specify two volumes in the workload template of which the second is bootable but the first is not, I don't think the instance will actually boot.  This is a launcher restriction and we need to enforce it here.